pkg/api/networkservice/mechanisms/memif/helpers.go

```diff
@@ -29,14 +27,13 @@ type Mechanism struct {
 	*networkservice.Mechanism
 }
 
-// New returns *networkservice.Mechanism of type  memif using the given socketPath (file://socketPath)
-func New(socketPath string) *networkservice.Mechanism {
+// New returns *networkservice.Mechanism of type memif using the given netnsURL (file:///proc/${pid}/ns/net)
+func New(netnsURL string) *networkservice.Mechanism {
```

Comment:
```suggestion
func FromNetNsURL(netnsURL string) *networkservice.Mechanism {
```

This would let us keep the existing 'New' and ease the transition.